AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_AllParamDatatypes_CreationSuccess.py

```diff
@@ -92,11 +91,13 @@ async def run_test(self):
         # 1) Open Asset Editor
         # Initially close the Asset Editor and then reopen to ensure we don't have any existing assets open
         general.close_pane(ASSET_EDITOR_UI) # this doesn't close a file that was previously open if you had just run a test that created an asset
-        general.open_pane(ASSET_EDITOR_UI)
-        helper.wait_for_condition(lambda: general.is_pane_visible(ASSET_EDITOR_UI), WAIT_TIME_3)
+        tools.open_asset_editor()
+        tools.initialize_editor_object(self)
 
-        # 2) Initially create new Script Event file with one method
-        tools.initialize_asset_editor_qt_objects(self)
+        # 2) Initialize asset editor, script canvas and create new Script Event file with one method
+        tools.initialize_asset_editor_object(self)
+        tools.open_script_canvas()
+        tools.initialize_sc_editor_objects(self)
         action = pyside_utils.find_child_by_pattern(self.asset_editor_menu_bar, {"type": QtWidgets.QAction, "text": SCRIPT_EVENT_UI})
         action.trigger()
```

Comment:
Let's create a helper function here. This is cryptic enough that it would be nice to wrap it in a function with a meaningful name that gives us a way to know what's going on without decoding the steps.

Reply:
converted much of this to helper functions in the tools file.  also made some of the looping behavior in step 7 easier to understand